code-challenges.test.js

```diff
@@ -14,46 +14,272 @@
+*/
+
+// Creating a test object
+// const object4 = { number: 5 };
+// Assigning the value of a new variable to ONLY the value of the key:value pair using Object.values()
+// const valueOnly = Object.values(object4);
+// Checking the new variable with a console.log
+// console.log(valueOnly);
+// Output: [5]
+
+/*
+Success! Now that I know how to access the value of any object, I can move forwards with creating the function.
+
+--------- FUNCTION INFORMATION:
+Describe what the function will do: The function will accept an object, then use comparison operators to determine whether the value of that object is divisible by 3.
+Function Name: isObjectEven
+Function Parameter(s): 1, object
+--------- INPUT / OUTPUT:
+input 1: object
+output 1: "value is divisible by three"
+output 2: "value is not divisible by three"
+--------- STEPS/METHODS:
+1. Declare a function called isObjectEven that will accept an object as a parameter.
+2. Create an empty variable (tempVar) that will hold an array containing only the value of the key:value pair using Object.values().
+3. Use a comparison operator to determine if the value of the resuting array is divisible by three.
+    3-1. If it IS divisible, return: `${tempVar} is divisible by three`
+    3-2. If it is NOT divisible, return: `${tempVar} is not divisible by three`
+    3-3. Include a catch-all error message as the else.
+4. Test!
+*/
+
+const isObjectEven = (object) => {
+  let tempVar = Object.values(object);
+  if (tempVar % 3 === 0) {
+    return `${tempVar} is divisible by three`;
+  } else if (tempVar % 3 !== 0) {
+    return `${tempVar} is not divisible by three`;
+  } else {
+    return "Oops! We've hit a snag.";
+  }
+};
+// Result: Test Suites: 1 passed, 1 total
+
+// So, even though this code DEFINITELY works, after helping another classmate solve the same problem, I realized that I could have also solved this by simply using object.number. I was trying to think in broad terms about needing to convert the value of the key, possibly without knowing (or needing to know) what that key name was. So, just for fun, I'm going to make another version here using that method as well, then test and comment it out.
+
+// const isObjectEven = (object) => {
+//   if (object.number % 3 === 0) {
+//     return `${object.number} is divisible by three`;
+//   } else if (object.number % 3 !== 0) {
+//     return `${object.number} is not divisible by three`;
+//   } else {
+//     return "Oops! We've hit a snag.";
+//   }
+// };
+// Result: Test Suites: 1 passed, 1 total
+
+// I'm happy with the result of my original function, so that's the one I'm going to leave un-commented. I think it's following the theory of making functions as reusable as possible, so I'm happy I did the research to discover object methods!
```

Comment:
Awesome!  It is always good to try multiple ways as this will help you discover tools that may be useful in the future.  Just note that Object.values will give you ALL of your values in the object which has some great use cases, but for this case might give unexpected results if other keys are in the object with a different data type.